src/QMCWaveFunctions/Fermion/MultiDiracDeterminantCalculator.h

```diff
@@ -2,9 +2,10 @@
 // This file is distributed under the University of Illinois/NCSA Open Source License.
 // See LICENSE file in top directory for details.
 //
-// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+// Copyright (c) 2019 QMCPACK developers.
```

Comment:
Please do not change this line. The copyright mark of this file started since 2016 instead of 2019

Reply:
This is exactly the opposite of what ORNL and CSCS legal told us on DCA, they told us that anytime a file was touched it should be updated to the current year.

Reply:
Then I was wrong. Good to know and we  need to let people know this.

Reply:
Updating the copyright year to the current year upon editing a file was the advice in place when I left Intel.